### ISSUE TITLE
Cython dependency still broken in release 0.16.X automatic installations

### ISSUE THREAD

**Issue by ddkinnamon** [arq5x/gemini#504]
This is a continuation of #501. The previous fix causes a new error for both 0.16.1 and 0.16.2:
```
Error: No packages found in current linux-64 channels matching: cython 0.22.0 py27_0

You can search for this package on Binstar with

    binstar search -t conda cython 0.22.0 py27_0
```
It looks like 0.22.0 is not a valid Cython version string (should be 0.22).

**Comment by ddkinnamon** [comment]
It actually looks like this is related to another issue affecting several releases. The last versioned install that worked for me was 0.12.2. Also, I can install using "latest" as the version, although there are several unit test failures. I tried to install 0.14.0 because it does not require bcolz, and, strangely enough, the versioned install script tried to install bcolz and died with the same error because 0.14.0 only requires Cython 0.20.1 in its `requirements_conda.txt`. It looks to me like the problem might be that `versioning/0.XX.X/requirements_pip.txt` reads `gemini>=0.XX.X`, rather than `gemini==0.XX.X` for all versions after 0.12.2. That may be causing pip to install the *latest* release (now 0.16.2) and its additional dependencies rather than the specified release. Note that `requirements.txt` at the repository root also specifies `gemini==0.16.2` and `cython>=0.22`, which may be why "latest" works.

**Comment by brentp** [comment]
@ddkinnamon could you try it ? I just changed them all to `==` as you suggested.

**Comment by ddkinnamon** [comment]
Thanks! I successfully installed 0.14.0 and 0.16.2. The 0.15.X series still has a broken Cython dependency (it is 0.20.1 in the requirements_conda.txt). I also resolved most of the unit test problems in 0.16.2 but opened another issue (#508) relating to that.
